src/s2/s2polyline.h

```diff
@@ -47,6 +47,10 @@ class S2Polyline final : public S2Region {
   // or Decode().
   S2Polyline();
 
+// S2Polyline is movable, but only privately copyable.
```

Comment:
Sorry, I wasn't clear -- with normal two-space indentation.

Reply:
Ah, crap. My bad, the lack of indentation was unintentional.